# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/RetainedDisplayListBuilder.h
# Commit: a69c40accbf0
# Full Hash: a69c40accbf099d041417ce4da9b82829f044167
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2023-12-06 21:30:12
# Regressor Bug: 1860328
# File Overlap Count: 2
# Description:
#   Bug 1860328 - Fix a bug where caret misses when dragging and dropping r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192377
# ==============================================================================

diff -r c9558e7c5c2d -r a69c40accbf0 layout/painting/RetainedDisplayListBuilder.h
--- a/layout/painting/RetainedDisplayListBuilder.h	Wed Dec 06 16:17:57 2023 +0000
+++ b/layout/painting/RetainedDisplayListBuilder.h	Wed Dec 06 16:29:18 2023 +0000
@@ -270,7 +270,6 @@
 
   nsDisplayListBuilder mBuilder;
   RetainedDisplayList mList;
-  WeakFrame mPreviousCaret;
   RetainedDisplayListMetrics mMetrics;
   RetainedDisplayListData mData;
 };